src/Stratis.Bitcoin.Features.MemoryPool/BlocksDisconnectedSignaled.cs

```diff
@@ -49,6 +49,12 @@ private async Task AddBackToMempoolAsync(Block block)
 
                 foreach (Transaction transaction in block.Transactions)
                 {
+                    if(transaction.IsCoinBase)
+                        continue;
+
+                    if(transaction.IsCoinStake)
```

Comment:
Spacing

Reply:
You don't want to have this in general case, consider using isprotocoltransaction equivalent